build.gradle

```diff
@@ -30,35 +33,35 @@ allprojects {
     project.ext {
         constants = [
                 minSdkVersion: 21,
-                targetSdkVersion: 31,
-                compileSdkVersion: 31,
+                targetSdkVersion: 33,
+                compileSdkVersion: 33,
         ]
 
-        buildToolVersion          =    '30.0.0'
-        kotlin_version            =    '1.6.20'
-        appCompatVersion          =    '1.3.0'
-        materialVersion           =    '1.3.0'
-        supportVersion            =    '28.0.0'
-        recyclerViewVersion       =    '1.1.0'
-        junitVersion              =    '4.12'
-        extJunitVersion           =    '1.1.2'
-        espressoVersion           =    '3.3.0'
-        roboelectricVersion       =    '4.4'
-        constraintLayoutVersion   =    '2.0.4'
+        androidxRunner            =    '1.5.0'
+        androidxCoreKt            =    '1.10.1'
+        appCompatVersion          =    '1.6.1'
+        buildToolVersion          =    '31.0.0'
+        composeActivityVersion    =    '1.7.2'
+        composeBomVersion         =    '2023.06.01'
+        composeTestVersion        =    '1.4.3'
+        composeVersion            =    '1.4.7'
```

Comment:
Do we need this when bom is used?

Reply:
composeVersion is used for compiler version
composeTestVersion is used for compose test library version, it isn't working with bom

Reply:
Rename composeVersion  to composeCompilerVersion